osmomath/math_test.go

```diff
@@ -241,3 +242,78 @@ func TestOrderOfMagnitudeWithoutLog(t *testing.T) {
 		})
 	}
 }
+
+func TestLegacyMultiMulMut(t *testing.T) {
+	type testCase struct {
+		base Dec
```

Comment:
For more rigorous tests that'll detect regressions and stand battering of time, I highly recommend statically posting up the results you expected as want, instead of dynamically computing them in the loop down below then comparing. That is because any subtle changes to MutMut or the other code being tested won't be caught.